http://gitweb.freedesktop.org/?p=xorg/xserver.git;a=commit;h=a23b0b069cac8a48e2b306b2095515d75f647705
Typo fix.
	author  Adam Jackson <ajax@benzedrine.nwnk.net>
	Mon, 12 Feb 2007 22:50:00 +0000 (17:50 -0500)
	committer  Adam Jackson <ajax@benzedrine.nwnk.net>
	Mon, 12 Feb 2007 22:50:00 +0000 (17:50 -0500)
	commit  a23b0b069cac8a48e2b306b2095515d75f647705
	tree  44d98fa9da8b879e4d610a9fc484804a418d217c  tree
	parent  d21c95f80bdba2f29eedd57fb0b00e580391c08e  commit | commitdiff
Typo fix.

--- xorg-server-1.2.0.orig/hw/xfree86/os-support/xf86_libc.h	2007-03-18 21:43:21.000000000 +0100
+++ xorg-server-1.2.0/hw/xfree86/os-support/xf86_libc.h	2007-03-19 00:31:52.000000000 +0100
@@ -421,21 +421,21 @@
 #undef tan
 #define tan(d)			xf86tan(d)
 #undef tmpfile
 #define tmpfile()		xf86tmpfile()
 #undef tolower
 #define tolower(i)		xf86tolower(i)
 #undef toupper
 #define toupper(i)		xf86toupper(i)
 #undef ungetc
 #define ungetc(i,FP)		xf86ungetc(i,FP)
-#undef vfprinf
+#undef vfprintf
 #define vfprintf(p,f,a)		xf86vfprintf(p,f,a)
 #undef vsnprintf
 #define vsnprintf(s,n,f,a)	xf86vsnprintf(s,n,f,a)
 #undef vsprintf
 #define vsprintf(s,f,a)		xf86vsprintf(s,f,a)
 /* XXX Disable assert as if NDEBUG was defined */
 /* Some X headers defined this away too */
 #undef assert
 #define assert(a)		((void)0)
 #undef HUGE_VAL